src/color.js

```diff
@@ -83,9 +83,8 @@ module.exports = class Color {
 
   // Essential: Returns a {String} in the form `'#abcdef'`.
   toHexString() {
-    return `#${numberToHexString(this.red)}${numberToHexString(
-      this.green
-    )}${numberToHexString(this.blue)}`;
+    let colorObj = ParsedColor.rgb(this.red, this.green, this.blue);
+    return colorObj.hex();
```

Comment:
```suggestion
    return (colorObj.hex()).toUpperCase();
```

Looks like originally Atom would convert these as uppercase parts of text, but the `colors` package does not. This should be enough to resolve the issue